SpatialGDK/Documentation/content/workers/set-up-offloading.md

```diff
@@ -2,18 +2,19 @@
 
 # Set up your game feature for offloading
 
-To set up your game feature for offloading, in addition to configuring your Unreal game server, you also need to start an Unreal process for your offloaded worker types, both locally in PIE and in the cloud. In particular, you specify the number of offloaded workers to start and their attributes in the Unreal Editor, and configure Actor groups to set the authority configuration for the Actors.
+To set up your game feature for offloading, in addition to configuring your Unreal game server, you also need to start an Unreal process for your offloaded worker types, both locally in PIE and in the cloud. In particular, you specify the number of offloaded servers to start and their attributes in the Unreal Editor, and configure Actor groups to set the authority configuration for the Actors.
 
 Before you start, review the following offloading workflows. You can check each step for detailed information:
 
 1. Create a [worker configuration file]({{urlRoot}}/content/glossary#worker-configuration-file): You create a worker configuration file for the new offloaded worker type.
+    **Notes**:
+    - If you launch a local deployment through PIE within the editor, the Unreal GDK adds the worker configuration file for your new offloaded worker type, so you can skip this step.
+    - If you launch a local deployment through PIE within the editor, the Unreal GDK adds the worker configuration file for your new offloaded worker type, so you can skip this step.
+2. Specify the launch configuration: 
```

Comment:
Is this duplicated?

    - If you launch a local deployment through PIE within the editor, the Unreal GDK adds the worker configuration file for your new offloaded worker type, so you can skip this step.
    - If you launch a local deployment through PIE within the editor, the Unreal GDK adds the worker configuration file for your new offloaded worker type, so you can skip this step.